lib/dest/write-contents/index.js

```diff
@@ -38,26 +40,13 @@ function writeContents(opt) {
     // This is invoked by the various writeXxx modules when they've finished
     // writing the contents.
     function onWritten(writeErr) {
-      if (isErrorFatal(writeErr)) {
+      if (fo.isOverwriteError(writeErr, file.flag)) {
```

Comment:
That's why we want the negative version of it, right?  `isNotOverwriteError`?  I typically shy away from inverted function names but is seems clearer here.